subsys/mgmt/hawkbit/Kconfig

```diff
@@ -33,9 +33,17 @@ config HAWKBIT_POLL_INTERVAL
 	  This time interval is zero and 43200 minutes(30 days). This will be overridden
 	  by the value configured in the settings of the hawkBit server.
 
+config HAWKBIT_AUTOHANDLER
+	bool "hawkBit autohandler"
+	select EVENTS
+	default y
+	help
+	  Activate autohandler to handle the update process automatically.
+
 config HAWKBIT_SHELL
 	bool "hawkBit shell utilities"
 	depends on SHELL
+	depends on HAWKBIT_AUTOHANDLER
```

Comment:
> And fore the ones, who don't want to use this autohandler, they can, if this PR gets merged, deactivate the included autohandler.

Does this dependency means that for user who doesn't want to use the bundled autohandler, they lose access to shell for updating the configurations as well?

Reply:
Well yes at least when they don't activate the Kconfig for the autohandler. They could still choose to include the autohandler and don't use it theirself.